packages/core/upload/server/content-types/file/schema.js

```diff
@@ -108,5 +108,30 @@ module.exports = {
       columns: ['folder_path'],
       type: null,
     },
+    {
+      name: `upload_files_created_at_index`,
+      columns: ['created_at'],
+      type: null,
+    },
```

Comment:
Read a conversation somewhere about indexes in strapi.
If I recall correctly, this is not ready for the public yet, but I imagine you can still use this syntax in any of your content-types right?

What work would be needed to make it public? Specify the list of fields you want to index to automatically generate a index name?

Reply:
Yes exactly :) and to convert the field name to the database column name (`createdAt` => `updatedAt`). Also some thinking to have about columns that don't appear in the user schema (like createdAt and updatedAt)